### DISCUSSION TITLE
Metal renderer locks fps to 240 when moving mouse during gameplay/replay?

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#22996] [Q&A] [answered]
As seen in the video below and as titled, this is much more obvious when not recording (400-500fps -> 240fps), possibly related to #22962?

https://user-images.githubusercontent.com/7279624/227811163-752def73-0ca1-4818-be28-6efe0a2e147e.mp4

[database.log](https://github.com/ppy/osu/files/11073325/database.log)
[input.log](https://github.com/ppy/osu/files/11073326/input.log)
[network.log](https://github.com/ppy/osu/files/11073327/network.log)
[performance.log](https://github.com/ppy/osu/files/11073328/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11073329/runtime.log)

**Comment by peppy** [comment, marked as answer]
I doubt it's "locking". It's just running slower when things are happening / updating.

**Reply by integerrr** [reply]
This is perfectly playable as is but am I right to assume more optimisation to come this front?

**Reply by peppy** [reply]
Yes, pleas read the changelo
